packages/editor/__tests__/specs/authorEditPanel.tsx

```diff
@@ -35,7 +35,13 @@ describe('Author Edit Panel', () => {
             author: {
               __typename: 'Author',
               id: 'fakeId2',
-              name: 'Douglas Cole'
+              name: 'Douglas Cole',
+              slug: '',
```

Comment:
Absence of these fields threw an error after updating Apollo. Probably bug by the library itself: https://github.com/apollographql/apollo-client/issues/8677